ci/L0_backend_vllm/vllm_backend/vllm_backend_test.py

```diff
@@ -163,6 +164,17 @@ def test_exclude_input_in_output_true(self):
             expected_output=expected_output,
         )
 
+    def test_ensemble_model(self):
```

Comment:
This is going to break `EXPECTED_NUM_TESTS=6`. 

You could update it to `EXPECTED_NUM_TESTS=7`, but I think it's easier to just append the ensemble to the other test that is testing model loading here: https://github.com/triton-inference-server/vllm_backend/blob/78bf0fbaac9c4e1e40e717f0411ff6c48b5d6afe/ci/L0_backend_vllm/vllm_backend/vllm_backend_test.py#L55-L59

Also use `vllm_load_test` model inside the ensemble instead of `vllm_opt` for the same reason.

Reply:
The tests documents some expected behavior arond `vllm_opt` staying alive for the duration of the test: https://github.com/triton-inference-server/vllm_backend/blob/78bf0fbaac9c4e1e40e717f0411ff6c48b5d6afe/ci/L0_backend_vllm/vllm_backend/test.sh#L53-L57

So I think it's best not to mess with it in your new test and make use of `vllm_load_test` instead.